analyzer/os/ubuntu/ubuntu.go

```diff
@@ -18,9 +20,22 @@ func init() {
 	analyzer.RegisterAnalyzer(&ubuntuOSAnalyzer{})
 }
 
-const version = 1
+const (
+	version            = 1
+	ubuntuConfFilePath = "etc/lsb-release"
+	esmConfFilePath    = "var/lib/ubuntu-advantage/status.json"
+	esmServiceName     = "esm-infra"
+	esmStatusEnabled   = "enabled"
+	esmVersionSuffix   = "ESM"
+)
 
-var requiredFiles = []string{"etc/lsb-release"}
+var (
+	osVersion     = ""
```

Comment:
is package scope for `osVersion` correct? Can it be used more than once in one fanal invocation?

Reply:
`osVersion` value is correctly preserved for all calls to `Analyze` function.

Reply:
I was wrong. 
When scanning more than 1 image, we got the wrong `osVersion`.
I used `context` to save the osVersion parts for each image. 
Can you take a look, when you will time.